src/main/java/org/sqlite/SQLiteLimits.java

```diff
@@ -0,0 +1,27 @@
+package org.sqlite;
+
+public enum SQLiteLimits {
+
+    SQLITE_LIMIT_LENGTH(0),
+    SQLITE_LIMIT_SQL_LENGTH(1),
+    SQLITE_LIMIT_COLUMN(2),
+    SQLITE_LIMIT_EXPR_DEPTH(3),
+    SQLITE_LIMIT_COMPOUND_SELECT(4),
+    SQLITE_LIMIT_VDBE_OP(5),
+    SQLITE_LIMIT_FUNCTION_ARG(6),
+    SQLITE_LIMIT_ATTACHED(7),
+    SQLITE_LIMIT_LIKE_PATTERN_LENGTH(8),
+    SQLITE_LIMIT_VARIABLE_NUMBER(9),
+    SQLITE_LIMIT_TRIGGER_DEPTH(10),
+    SQLITE_LIMIT_WORKER_THREADS(11);
+
+    private final int id;
+
+    SQLiteLimits(int id) {
```

Comment:
If there is no specific reason, we can make the constructor private.

Reply:
Yep it can be private. No reason it couldn’t.

Reply:
ok. Please do that. I'll merge this PR after that

Reply:
That's done ^^